rpcs3/Emu/Cell/lv2/sys_hid.h

```diff
@@ -1,6 +1,37 @@
 #pragma once
 
 #include "Emu/Memory/vm_ptr.h"
+#include "Emu/Memory/vm_var.h"
+#include "sys_sync.h"
```

Comment:
these includes seem to be unused?

Reply:
sys_sync seems its unused, vm_var is def used

Reply:
but is it used in the header ?

Reply:
actually get a ton of cannot overload function by return type alone without them

Reply:
might just be Vs being weird

Reply:
move the vm_var include to the cpp and replace the sys_sync include here with ErrorCode include

Reply:
DO EET

Reply:
Done <3